sips/sip-150.md

```diff
@@ -0,0 +1,99 @@
+---
+sip: 150
+title: Issuance Debt Adjustment Truncation Fix
+status: Draft
+author: Anton Jurisevic (@zyzek)
+discussions-to: https://research.synthetix.io
+created: 2021-06-16
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+This corrects a potentially-exploitable problem whereby
+issuer's debt balances immediately after issuing synths are less than the sUSD they have minted.
+
+
+## Abstract
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+
+Since the EtherWrapper cap was raised, calculation of the excluded debt component of partial debt updates has been
+interfering with incrementing the cached debt number upon issuance of synths.
+This SIP resolves the issue by not computing the excluded debt component during sUSD issuance.
+
+
+## Motivation
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is innaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+
+The attack surface is limited by the size of a user's wallet, since the available profit is at most the user's issued
+fraction of the debt pool. The burn lock ([SIP 40](https://sips.synthetix.io/sips/sip-40)) also makes this
+difficult to take advantage of in practice.
+However, if a debt snapshot is not performed quickly enough, an issuer could make free profit
+by burning their newly-issued sUSD, freeing up all their staked SNX while leaving some sUSD left over.
+
+
+## Specification
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+The `DebtCache._updateCachedSynthDebtsWithRates` function is called during issuance to update the cached debt with the
```

Comment:
I would add a bit more context here on where this change was introduced and why - "in SIP-136, the debt calculation was amended to correctly exclude debt from total system debt. This included an update to the cached debt recalculation in `_updateCachedSynthDebtsWithRates`, however the logic implemented was broken. It intended to subtract the excluded debt from total cached debt, however it in fact subtracted the excluded debt from the delta cached debt."